ui/src/components/AppView.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class AppDetailedView extends Component {
```

Comment:
Is it best practice to keep the class name the same as the file name?

Reply:
I don't think React is opinionated, at least lint didn't pick it up but I agree that we should name the components the same name as the file. It's done in all other files. It's probably a rename error, will fix it. :)